src/test/java/nextstep/courses/domain/SessionTest.java

```diff
@@ -0,0 +1,76 @@
+package nextstep.courses.domain;
+
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import java.util.Calendar;
+import java.util.Date;
+import nextstep.users.domain.NsUser;
+import nextstep.users.domain.NsUserTest;
+import org.junit.jupiter.api.Test;
+
+public class SessionTest {
+
+    @Test
+    void 강의는_강의_커버_이미지_정보와_수강_기간을_가진다() {
+        // given
+
+        Image image = new Image();
+
+        Period period = new Period();
+
+        // when
+        Session session = new FreeSession(image, period);
+
+        // then
+        assertThat(session.getImage()).isEqualTo(image);
+        assertThat(session.getPeriod()).isEqualTo(period);
+    }
+
+    @Test
+    void 강의는_유료강의와_무료강의로_나뉜다() {
+        // given
+        FreeSession freeSession = new FreeSession(new Image(), new Period());
+        PaySession paySession = new PaySession(new Image(), new Period(), 1, 1000L);
```

Comment:
상위 타입인 Session으로 선언하면 어떨까요?
```suggestion
        Session freeSession = new FreeSession(new Image(), new Period());
        Session paySession = new PaySession(new Image(), new Period(), 1, 1000L);
```